### PR TITLE
CSP to be consistent with preset name

### PR DESCRIPTION
**Pull request by haifeng-jin**
Change the `CSPDarkNetXBackbone` to `CSPDarkNetXLBackbone` to be consistent with the present name of `"csp_dark_net_xl"`.

### CODE REVIEW

**Review by LukeWood — APPROVED** [review]

File: keras_cv/models/backbones/csp_darknet/csp_darknet_backbone.py [hunk 1]
```diff
@@ -387,7 +387,7 @@ def presets(cls):
     ),
 )
 setattr(
-    CSPDarkNetXBackbone,
+    CSPDarkNetXLBackbone,
     "__doc__",
     ALIAS_DOCSTRING.format(
         name="X",
```
Comment (ianstenbit) [inline keras_cv/models/backbones/csp_darknet/csp_darknet_backbone.py:393]: This should become XL to keep the docstring correct

File: keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_test.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ def test_create_backbone_model_with_level_config(self):
         ("S", csp_darknet_backbone.CSPDarkNetSBackbone),
         ("M", csp_darknet_backbone.CSPDarkNetMBackbone),
         ("L", csp_darknet_backbone.CSPDarkNetLBackbone),
-        ("X", csp_darknet_backbone.CSPDarkNetXBackbone),
+        ("X", csp_darknet_backbone.CSPDarkNetXLBackbone),
```
Comment (ianstenbit) [inline keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_test.py:134]: Probably also worth making this string "XL", although it doesn't really matter a lot

**Review by ianstenbit — APPROVED** [review]
LG -- two little nits.

Thank you!

File: keras_cv/models/backbones/csp_darknet/csp_darknet_backbone.py (hunk 1 above)
Comment (haifeng-jin) [reply keras_cv/models/backbones/csp_darknet/csp_darknet_backbone.py]: Done.

File: keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_test.py (hunk 2 above)
Comment (haifeng-jin) [reply keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_test.py]: Done.

**Review by jbischof — APPROVED** [review]
Thank you!

**Pull request merged**